athena/Radius.tsx

```diff
@@ -95,22 +95,25 @@ function calculateRadius(
 
   while (!queue.isEmpty()) {
     const { cost: parentCost, vector } = queue.poll()!;
-    if (closed[map.getTileIndex(vector)]) {
+    const index = map.getTileIndex(vector);
+    if (closed[index]) {
       continue;
     }
+    closed[index] = true;
```

Comment:
This is the only line that matters. The rest of this PR just moves around variables.